### PR TITLE
Update inf-addservice-directive.md

### PR DESCRIPTION
**Pull request by limitedge**

### CODE REVIEW

File: windows-driver-docs-pr/install/inf-addservice-directive.md [hunk 1]
```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*[**,**_privilege-name_]...
+> [!NOTE]
+> This value can only be used for *Win32 Services* and is only available on Windows 7 and above.
+
+Each *privilege-name* in the list is the name of the privilege.  See [Privilege Constants (Windows)](/windows/win32/secauthz/privilege-constants) for a list of the privilegs names. For each privilege name, only the text name is required, *For Example* The privilege name should be written as "SetAuditPrivilege" but not SE_AUDIT_NAME.
```
Comment (jasonknichel) [inline windows-driver-docs-pr/install/inf-addservice-directive.md]: why "*For Example* The" (i.e. why is "For Example" italicized and why is "Example" and "The" capitalized?  Were you modeling it off of somewhere else on the page?  I would probably just do "For example, the" (note also the addition of the comma)

Comment (jasonknichel) [reply windows-driver-docs-pr/install/inf-addservice-directive.md]: oh, and I would turn the comma at the end of "required," into a period.

File: windows-driver-docs-pr/install/inf-addservice-directive.md [hunk 2]
```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
...
+
+For more information on service required privileges, see [SERVICE_REQUIRED_PRIVILEGES_INFO (Windows)](/windows/win32/api/winsvc/ns-winsvc-service_required_privileges_infow)
```
Comment (jasonknichel) [inline windows-driver-docs-pr/install/inf-addservice-directive.md:254]: Should have a period at the end of this sentence.

Comment (limitedge) [reply windows-driver-docs-pr/install/inf-addservice-directive.md]: Sure. This has been modified accordingly

File: windows-driver-docs-pr/install/inf-addservice-directive.md (hunk 1 above)
Comment (limitedge) [reply windows-driver-docs-pr/install/inf-addservice-directive.md]: Sure, will take care of it

File: windows-driver-docs-pr/install/inf-addservice-directive.md [hunk 3]
```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*[**,**_privilege-name_]...
+> [!NOTE]
+> This value can only be used for *Win32 Services* and is only available on Windows 7 and above.
+
+Each *privilege-name* in the list is the name of the privilege.  See [Privilege Constants (Windows)](/windows/win32/secauthz/privilege-constants) for a list of the privilegs names. For each privilege name, only the text name is required. For example, the privilege name should be written as "SeAuditPrivilege" but not SE_AUDIT_NAME.
```
Comment (jasonknichel) [inline windows-driver-docs-pr/install/inf-addservice-directive.md:252]: "is the name of the privilege" -> "is the name of a privilege that the service requires"
"privilegs names" -> "privilege names" (You have an 's' where there should be an 'e')

**Review by jasonknichel — APPROVED** [review]

**Comment by tedhudek** [comment]
Thanks for the PR, Paul, and welcome aboard.  Thanks for Jason for the review.

**Pull request merged**